### DISCUSSION TITLE
Steam equivalent?

### DISCUSSION THREAD

**Discussion by akeakeakeake** [victornpb/undiscord#669] [1. Questions & Answers]
Hey guys! Wondering if theres an undiscord equivalent for steam? Having owned my steam since 2013 when i was a child i have alot alot of cringe messages and discussions that also impede on my privacy. Is there any way to mass delete those?

**Comment by cratefarmer** [comment]
i dont know of any scripts, but if you want you can go to http://steamcommunity.com/my/commenthistory and review some cringy keywords from any and all comments, threads etc that you made. theres a nice handy filter there and you can delete comments or edit out threads (you cant delete threads)
